### PR TITLE
QS User Access - Submitting Party

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16838

*Description of changes:*
- Implements Submitting party and state management for QS User Access Flows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/store/state/state-model.ts [hunk 1]
```diff
@@ -379,7 +379,8 @@ export const stateModel: StateModelIF = {
       },
       phoneNumber: '',
       phoneExtension: ''
-    }
+    },
+    qsSubmittingParty: null
```
Comment (cameron-eyds) [inline ppr-ui/src/store/state/state-model.ts:383]: We don't need to initialize the entirety of the submitting party as this data is read only and wont be mutated.
As such, it won't need the reactive hooks created as other information that is collected.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1476-16t06ty4.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1476-16t06ty4.web.app

**Review by doug-lovett — APPROVED** [review]
Tres bon.

**Review by orelbn — APPROVED** [review]
Excellent

**Pull request merged**